app-emulation/vagrant/vagrant-1.9.7.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+USE_RUBY="ruby22 ruby23"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem eutils
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="http://vagrantup.com/"
+SRC_URI="https://github.com/mitchellh/vagrant/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+virtualbox"
+
+RDEPEND="${RDEPEND}
+	app-arch/libarchive
+	net-misc/curl
+	virtualbox? ( || ( app-emulation/virtualbox app-emulation/virtualbox-bin ) )"
+
+ruby_add_rdepend "
+	>=dev-ruby/childprocess-0.6.0
+	>=dev-ruby/erubis-2.7.0
+	>=dev-ruby/i18n-0.6.0:* <dev-ruby/i18n-0.8.0:*
```

Comment:
This can install two different versions in separate slots. I doubt that's what you want. Since there is no version older than `0.6.0`, you can just leave the `<` part.